config.md

```diff
@@ -7,304 +7,171 @@ This specification outlines the JSON format describing images for use with a con
-            </dt>
-            <dd>
-                Entries are in the format of <code>VARNAME="var value"</code>.
-                These values act as defaults and are merged with any specified when creating a container.
-            </dd>
-            <dt>
-                Entrypoint <code>array of strings</code>
-            </dt>
-            <dd>
-                A list of arguments to use as the command to execute when the container starts.
-		This value acts as a  default and is replaced by an entrypoint specified when creating a container. This field MAY be "null".
-            </dd>
-            <dt>
-                Cmd <code>array of strings</code>
-            </dt>
-            <dd>
-                Default arguments to the entrypoint of the container.
-		These values act as defaults and are replaced with any specified when creating a container.
-		If an <code>Entrypoint</code> value is not specified, then the first entry of the <code>Cmd</code> array should be interpreted as the executable to run. This field MAY be "null".
-            </dd>
-            <dt>
-                Volumes <code>struct</code>
-            </dt>
-            <dd>
-                A set of directories which should be created as data volumes in a container running this image. This field MAY be "null".
-                <p>
-                If a file or folder exists within the image with the same path as a data volume, that file or folder is replaced with the data volume and is never merged.
-                </p>
-		This JSON structure value is unusual because it is a direct JSON serialization of the Go type <code>map[string]struct{}</code> and is represented in JSON as an object mapping its keys to an empty object.
-		Here is an example:
-<pre>{
-    "/var/my-app-data/": {},
-    "/etc/some-config.d/": {}
-}</pre>
-            </dd>
-            <dt>
-                WorkingDir <code>string</code>
-            </dt>
-            <dd>
-                Sets the current working directory of the entrypoint process in the container.
-		This value acts as a default and is replaced by a working directory specified when creating a container.
-            </dd>
-        </dl>
-    </dd>
-    <dt>
-        rootfs <code>struct</code>
-    </dt>
-    <dd>
-        The rootfs key references the layer content addresses used by the image.
-	This makes the image config hash depend on the filesystem hash.
-        rootfs has two subkeys:
-
-        <ul>
-          <li>
-            <code>type</code> which MUST be set to <code>layers</code>.
-            Implementations MUST generate an error if they encounter a unknown value while verifying or unpacking an image.
-          </li>
-          <li>
-            <code>diff_ids</code> is an array of layer content hashes (<code>DiffIDs</code>), in order from bottom-most to top-most.
-          </li>
-        </ul>
-
-
-        Here is an example rootfs section:
-
-<pre>"rootfs": {
```

Comment:
There's a big example at the end which gets verified vs. the JSON Schema.  I think that's sufficient.